tracer/src/Datadog.Trace/PlatformHelpers/ContainerMetadata.cs

```diff
@@ -154,7 +154,7 @@ internal static bool TryGetInode(string path, out long result)
 
             try
             {
-                var statCommand = ProcessHelpers.RunCommand(new ProcessHelpers.Command("stat", $"--printf=%i {path}"));
+                var statCommand = ProcessHelpers.RunCommand(new ProcessHelpers.Command("stat", $"-c %i {path}"));
```

Comment:
We should probably add a unit tests for `TryGetInode()` to `ContainerMetadataTests.cs`. This test should fail on Alpine before this change and pass after the change.

Reply:
And in other Linux distros (not Alpine), it should pass before and after this change (confirming there is not regression).